NoComponentDrag.py

```diff
@@ -96,21 +101,31 @@ def set_direct_edit_drag_enabled(value):
 def get_direct_edit_drag_enabled():
     return fusion_drag_controls_cmd_def_.controlDefinition.isChecked
 
+    # This function was found to be the source of the multiple calls of "enable_cmd_created_handler"
+    # I noticed the update was being delayed but multiple sources can call check_environment leading to multiple updated being queued sequentially causing debugging to be a nightmare
+    # On stopping the multiple updates from queueing issue "NoComponentDrag breaks Insert -> Derive" was solved i assume because the final update was overriding the insert window
 def check_environment():
+    # If a check environment is already queued then skip adding another.
+    # IMPORTANT: May need to add code that removes the last placed check env and places another at the end of the queue if anything is queued in between calls of check env if errors occur
```

Comment:
Im basically saying that if this breaks anything that the original had working then its just because something is being queued after the first check env was called and all that needs to be done is to save a ref to the queue instance, delete it and let the code continue instead of returning which should ensure update is among the last to queue but thats only IF calling "delay" only moves it to the end of the current queue and things can be queued after if not then its not an issue unless other functions also call "delay" that can possibly change the order but I do not think that will be much of a problem and certainly not one I wanted to spend more time with someone else's code on since its hard enough to handle my own lol everything works for me and how i use fusion so I'm happy ^.^